lisa/notifiers/junit.py

```diff
@@ -28,6 +28,12 @@ class JUnitSchema(schema.Notifier):
     path: str = "lisa.junit.xml"
     # respect the original behavior, include subtest by default
     include_subtest: bool = True
+    # show passed case 'message' and 'stacktraces'
+    include_passed_messages: bool = False
```

Comment:
It doesn't need to be False by default, so the logic is more consistent for different types.

Reply:
I marked it false to ensure current output is not modified in any way. This is because i expect users to be parsing this output in their implementation of LISA. 
Marking passed and skipped flag as false by default allows us to maintain some level of compat with existing implementation while allowing customization for new implementations.

Reply:
If no scenario needs the stack trace for passed test cases, it doesn't need the flag.